code/game/objects/random/random.dm

```diff
@@ -1091,3 +1094,89 @@
 		/obj/item/weapon/gun/custom_ka/frameF/prebuilt01 = 1,
 		/obj/item/weapon/gun/custom_ka/frameF/prebuilt02 = 1
 	)
+
+/obj/random/vault_rig
+	name = "random rigsuit"
+	desc = "Contains a random rigsuit found in the vault."
+	icon = 'icons/obj/rig_modules.dmi'
+	icon_state = "breacher_rig"
+	spawnlist = list(
+		/obj/item/weapon/rig/ce/equipped = 1,
+		/obj/item/weapon/rig/combat = 0.5,
+		/obj/item/weapon/rig/diving = 0.1,
+		/obj/item/weapon/rig/ert/assetprotection/empty = 0.1,
+		/obj/item/weapon/rig/ert/janitor = 0.1,
+		/obj/item/weapon/rig/eva/equipped = 0.5,
+		/obj/item/weapon/rig/hazard = 1,
+		/obj/item/weapon/rig/hazmat = 1,
+		/obj/item/weapon/rig/military = 0.1,
+		/obj/item/weapon/rig/unathi = 1,
+		/obj/item/weapon/rig/vaurca/minimal = 0.1
+	)
+
+/obj/random/telecrystals
+	name = "random telecrystals"
+	desc = "Contains a random amount of telecrystals."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "telecrystal"
+	spawnlist = list(
+		/obj/item/stack/telecrystal{amount = 10} = 0.7,
+		/obj/item/stack/telecrystal{amount = 25} = 0.2,
+		/obj/item/stack/telecrystal{amount = 5} = 0.1
+	)
+
+/obj/random/bad_ai
+	name = "random evil AI module"
+	desc = "Contains a random evil AI module."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "std_mod"
+	spawnlist = list(
+		/obj/item/weapon/aiModule/antimov = 1,
+		/obj/item/weapon/aiModule/asimov = 1,
+		/obj/item/weapon/aiModule/purge = 1,
+		/obj/item/weapon/aiModule/quarantine = 1,
+		/obj/item/weapon/aiModule/freeform = 2,
+		/obj/item/weapon/aiModule/freeformcore = 1,
+		/obj/item/weapon/aiModule/oneHuman = 0.5,
+		/obj/item/weapon/aiModule/oxygen = 1,
+		/obj/item/weapon/aiModule/syndicate = 1
```

Comment:
NT wouldnt have a syndicate module in the vault

Reply:
Done.